controllers/userController.js

```diff
@@ -57,4 +57,64 @@ const emailCheck = async (req, res) => {
   }
 };
 
-module.exports = { signUp, signIn, emailCheck };
+const myPageShow = async (req, res) => {
```

Comment:
```suggestion
const getUserInfo = async (req, res) => {
```
api를 구현함에 있어서 각각의 메소드 naming은 정확하게 정해진 규칙은 없지만, CRUD에 맞춰서 (create, get, update, delete) `동작과 자원의 조합`으로 지어주시면 모든 function에서 일관된 naming으로 작성할 수 있습니다. 


다른 사람이 보기에도 보다 더욱 명확합니다.

Reply:
네 CRUD에 맞춰서 변수 네이밍 하겠씁니다.